### DISCUSSION TITLE
【有序列表顺序错误】列表中插入block、列表会重新编号

### DISCUSSION THREAD

**Discussion by expoli** [tangly1024/NotionNext#1952] [Q&A]
**描述bug**
有序列表顺序错误、未按 notion 编辑的顺序显示

**复现步骤**
一个有序列表、中间插入一个图片，后续在 notion 里面可以指定固定序号，但是博客渲染之后，发现序号直接从1开始

**期望的正常结果**
按照 notion 中指定的序号

**截图**
- 博客渲染页面
![图片](https://user-images.githubusercontent.com/31023767/211791242-501d775f-2cdb-4c92-a130-c99b67ecd199.png)
- notion 页面
![图片](https://user-images.githubusercontent.com/31023767/211791380-22738b70-2ada-463f-bba4-d6c3dfc64761.png)

**环境**

- 操作系统: [例如. iOS, Android, macOS, windows]
- 浏览器 [例如. chrome, safari, firefox]
- 版本 [e.g. 3.8.1]

**补充说明**
与问题相关的其它说明

**Comment by tangly1024** [comment]
react-notion-x 中的bug；

若需要在列表中插入信息，建议将插入的信息按下Tab缩进。
